packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useState } from 'react';
+import { Text } from '@deriv/components';
+import { Notifications as Announcement } from '@deriv-com/ui';
+import { StandaloneBullhornRegularIcon } from '@deriv/quill-icons';
+import { load } from '@deriv/bot-skeleton';
+import { save_types } from '@deriv/bot-skeleton/src/constants/save-type';
+import clsx from 'clsx';
+import { localize } from '@deriv/translations';
+import AnnouncementDialog from './announcement-dialog';
+import { ANNOUNCEMENTS } from './config';
+import './announcements.scss';
+import { DBOT_TABS } from 'Constants/bot-contents';
+import { IconAnnounce, MessageAnnounce, TitleAnnounce } from './announcement-components';
+
+type TAnnouncements = {
+    is_mobile?: boolean;
+    handleTabChange: (item: number) => void;
+};
+
+const Announcements = ({ is_mobile, handleTabChange }: TAnnouncements) => {
+    const [isAnnounceDialogOpen, setIsAnnounceDialogOpen] = useState(false);
+    const [isOpenAnnounceList, setIsOpenAnnounceList] = React.useState(false);
+    const [amountAnnounce, setAmountAnnounce] = useState({} as Record<string, boolean>);
+    const buttonRef = React.useRef<HTMLButtonElement>(null);
+    const wrapperRef = React.useRef<HTMLDivElement>(null);
```

Comment:
consistency in naming convention

```suggestion
    const [is_announce_dialog_open, setIsAnnounceDialogOpen] = useState(false);
    const [is_open_announce_list, setIsOpenAnnounceList] = React.useState(false);
    const [amount_announce, setAmountAnnounce] = useState({} as Record<string, boolean>);
    const button_ref = React.useRef<HTMLButtonElement>(null);
    const wrapper_ref = React.useRef<HTMLDivElement>(null);
```

Reply:
@sandeep-deriv In this case the useState's variables will have errors in the sonar cloud. Are you sure we want to see the errors in the sonar cloud?

Reply:
Let me check sonarCloud with @ali-hosseini-deriv , I think we already had a discussion to put this rule down